app-editors/scite/scite-4.2.1.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome2-utils toolchain-funcs xdg-utils
```

Comment:
gnome2-utils isnt used anymore, can be dropped. Few styling things, then it looks good to me!